src/docs/product/crons/getting-started.mdx

```diff
@@ -53,23 +53,24 @@ import requests
 
 headers = {'Authorization': 'DSN ___PUBLIC_DSN___'}
 monitor_id = '' # Write your monitor_id here
+org_slug = '' # Write your organization slug here
```

Comment:
I think there actually might be an auto dropdown thing for this in docs.


```suggestion
org_slug = '__ORG_SLUG__' # Write your organization slug here
```

Reply:
oooh nice, just like for dsn right?

Reply:
yup